test/e2e/suite/conformance/certificates/tests.go

```diff
@@ -691,6 +698,10 @@ func (s *Suite) Define() {
 		}, featureset.OnlySAN)
 
 		s.it(f, "should issue a certificate defined by an ingress with certificate field annotations", func(issuerRef cmmeta.ObjectReference) {
+			if s.HTTP01TestType != "Ingress" {
+				Skip("Skipping ingress-specific as non ingress HTTP-01 solver is in use")
+				return
+			}
```

Comment:
Why do we skip this? Is that due to haproxy-ingress requiring `certificateRef.group` non-empty?

Reply:
Added a comment to the Skip